src/components/templates/TalkShowStudio/components/Player/Player.scss

```diff
@@ -51,8 +59,15 @@
       user-select: none;
     }
 
-    &--on {
+    &--off {
+      visibility: visible;
```

Comment:
Why did we change this from `&--on` to `&--off`?

Reply:
This is a duplicate of https://github.com/sparkletown/sparkle/pull/1653#discussion_r659516418